src/components/Buttons/CircleButton.js

```diff
@@ -3,11 +3,11 @@
  *
  * @author [Dominic Cocchiarella ](https://github.com/docchia)
 */
-import colors       from '../../styles/colors'
-import responsive   from '../../styles/responsive'
-import themer       from '../../styles/themer'
-import zIndex       from '../../styles/zIndex'
-import { spacings } from '../../styles/spacing'
+import colors       from 'styles/colors'
+import responsive   from 'styles/responsive'
+import themer       from 'styles/themer'
+import zIndex       from 'styles/zIndex'
+import { spacings } from 'styles/spacing'
```

Comment:
If we do want to continue importing directly from the specific directory, this should mirror other style import conventions and be:

```
import spacings from 'styles/spacing'
```